### PR TITLE
Initial working version

### PR DESCRIPTION
**Pull request by mvdriel**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# tasks file for rsyslog
+---
+- name: install
+  apt:
```
Comment (tersmitten) [inline tasks/main.yml:4]: `sudo`?

File: tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    validate: "rsyslogd -N1 -f %s"
+    owner: root
+    group: root
+    mode: 0440
+  with_dict: rsyslog_rsyslog_d_files
+  tags: [configuration, rsyslog, rsyslog-configuration, rsyslog-configuration-rsyslog-d]
+
+- name: update global configuration file
+  template:
+    src: etc/rsyslog.conf.j2
```
Comment (tersmitten) [inline tasks/main.yml:24]: `/etc/rsyslog.conf`?

File: templates/etc/rsyslog.conf.j2 [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+#			For more information see
+#			/usr/share/doc/rsyslog-doc/html/rsyslog_conf.html
+#
+#  Default logging rules can be found in /etc/rsyslog.d/50-default.conf
+
+
+#################
+#### MODULES ####
+#################
+
```
Comment (tersmitten) [inline templates/etc/rsyslog.conf.j2:13]: What could be the configuration format for `$ModLoad`?

File: tasks/main.yml (hunk 1 above)
Comment (mvdriel) [inline tasks/main.yml:4]: Oops, fixed

File: tasks/main.yml (hunk 2 above)
Comment (mvdriel) [inline tasks/main.yml:24]: Fixed

File: templates/etc/rsyslog.conf.j2 (hunk 3 above)
Comment (mvdriel) [inline templates/etc/rsyslog.conf.j2:13]: A hash with the name of the module as key and the configuration options for that module as an array with name/value pairs in the values?

**Pull request merged**